mint.json

```diff
@@ -102,6 +102,7 @@
           "pages": [
             "send-data/cloudfront",
             "send-data/cloudwatch",
+            "send-data/aws-dot",
```

Comment:
I believe this should be `adot` not `dot`

Reply:
Thanks for this. The official name is `AWS Distro for OpenTelemetry` so we either abbreviate it as `AWS DOT` or `ADOT`. But `AWS ADOT` isn't correct.

Reply:
makes sense, but from the first look just `dot` feels weird, I haven't seen it written anywhere like that (in their docs). In this [page](https://aws-otel.github.io/docs/introduction) they always write `ADOT`. Maybe the url can just do `/send-data/ADOT` without aws part?